tests/tests/swfs/avm2/bitmap_constr/Test.as

```diff
@@ -0,0 +1,31 @@
+﻿package {
+	public class Test {
+	}
+}

+import flash.display.Bitmap;
+
+trace("///var b1 = new Bitmap();");
+var b1 = new Bitmap();
+
+trace("///b1.bitmapData;");
trace(b1.bitmapData);
+
+trace("///b1.pixelSnapping;");
+trace(b1.pixelSnapping);

+trace("///b1.smoothing;");
+trace(b1.smoothing);


```

Comment:
These `^M` control sequences look a bit messy, they are in several .as files.

Reply:
I would really like to know why Animate CC started doing this.